server/views/pages/list.njk

```diff
@@ -65,11 +65,19 @@
 
   <div class="row {{ {s:2, m:2, l:2} | spacingClasses({padding: ['top']}) }} {{ {s:3, m:3, l:3} | spacingClasses({padding: ['bottom']}) }} ">
     <div class="container">
-      <div class="grid">
-        <div class="grid__cell">
-          {% componentV2 'pagination', pagination %}
+      {% if moreLink %}
+        <div class="grid">
+          <div class="grid__cell" style="text-align: right">
+            <a href="{{moreLink}}">More articles from the archive</a>
```

Comment:
How about using the more-info-link component for consistency?

`{% componentV2 'more-info-link', {name: 'More articles from the archive', url: moreLink} %}`